### PR TITLE
US11 // parts - highlight active filters

### PR DESCRIPTION
**Pull request by matschi3**
highlighted the active filter for each of the filter-slider

[US11](https://github.com/matschi3/capstone/issues/20)
[Deployment](https://capstone-git-feature-us11highlight-active-filters-matschi3.vercel.app/)

click a filter and see the highlighting. (just a little bit)

### CODE REVIEW

File: components/PartsList/index.js [hunk 1]
```diff
@@ -30,31 +32,86 @@ export default function PartsList({ parts }) {
...
+              value: "figur",
+              name: "figur",
+              funct2: setActiveCategoryFilter,
+              value2: "figur",
+            },
           ]}
+          activeFilter={activeCategoryFilter}
         />
         <StyledFilter
           sets={[
```
Comment (actyvystom) [inline components/PartsList/index.js:77]: A smart idea to populate your filters! 🚀 
I suggest some naming changes to shed some light on it for external reviewers (and for yourself in the future when you review your code again after some time has passed 😉 )

```js
{
    primaryFunction: setStatusFilter,
    primaryValue: "alle",
    filterName: "alle",
    secondaryFunction: setActiveStatusFilter,
    secondaryValue: "alle",
}
```

**Review by actyvystom — COMMENTED** [review]
Nice work! 🚀 
See some tiny suggestions on naming.
Cheers,
Thomas

File: components/PartsList/index.js (hunk 1 above)
Comment (matschi3) [reply components/PartsList/index.js]: Thanks for your advise. I will stay consistent in naming from now on ✌🏼   
it looks muss better now

**Comment by matschi3** [comment]
// reworked naming

**Pull request merged**